docs/aws.md

```diff
@@ -2,322 +2,380 @@
   <img src="img/k8s-aws.png"> </image>
 </p>
 
-
-
 # Getting Started
 
 ## Install kops
 
-From Homebrew:
-
-```bash
-brew update && brew install --HEAD kops
-```
-
-From Source:
-
-```bash
-go get -d k8s.io/kops
-cd ${GOPATH}/src/k8s.io/kops/
-git checkout release
-make
-```
-
-See our [installation guide](build.md) for more information
+Before we can bring up the cluster we need to [install the CLI tool](install.md) `kops`.
 
 ## Install kubectl
 
-It is a good idea to grab a fresh copy of `kubectl` now if you don't already have it.
-
-#### OS X
-
-```
-brew install kubernetes-cli
-```
+In order to control Kubernetes clusters we need to [install the CLI tool](install.md) `kubectl`.
 
 #### Other Platforms
 
 * [Kubernetes Latest Release](https://github.com/kubernetes/kubernetes/releases/latest)
 
 * [Installation Guide](http://kubernetes.io/docs/user-guide/prereqs/)
 
-
 ## Setup your environment
 
-### Setting up a kops IAM user
+### AWS
 
+In order to correctly prepare your AWS account for `kops`, we require you to
+install the AWS CLI tools, and have API credentials for an account that has
+administrator level privileges.
```

Comment:
I don't think we need full admin, and this might be a red flag for some new users.. Can we point to https://github.com/kubernetes/kops/blob/master/docs/aws.md#create-an-iam-user-for-kops for the privileges here? I went ahead and merged @jasondewitt's PR on this doc already, so this should be in now.